lib/dynflow/semaphores/action_input.rb

```diff
@@ -0,0 +1,21 @@
+module Dynflow
+  module Semaphores
+    class ActionInput < Stateful
```

Comment:
Is the `ActionInput` the best name for the thing? It might be missleading and someone might thing it has something to do with `action.input`